docs/ff-integrations/ads/admob.md

```diff
@@ -427,53 +279,22 @@ To load the ad on page load:
 </div>
 <p></p>
 
-### 4. Display Ad
+## Best Practices
 
-In this step, you will show the ad plus check if a user has dismissed the ad. If so, you will load the new ad and then move to the next page.
+To maximize the effectiveness of AdMob ads in your app while maintaining a positive user experience and complying with AdMob policies, follow these overall best practices:
 
-The steps to display an interstitial ad are as follows:
+1. **Use Test Ads During Development**: Always enable Test Ads during development to avoid invalid traffic and protect your AdMob account from being flagged or banned.
```

Comment:
They should be bullet points and not numbered. Fix for all the best practices below this.

Reply:
ok, fixed.